scikeras/wrappers.py

```diff
@@ -100,62 +162,73 @@ def __init__(
         self.validation_split = validation_split
         self.shuffle = shuffle
         self.run_eagerly = run_eagerly
+        self.epochs = epochs
 
         # Unpack kwargs
         vars(self).update(**kwargs)
 
+        # Check for deprecated APIs
+        if self.model is None and build_fn is not None:
+            self.model = build_fn
+            warnings.warn(
+                "`build_fn` will be renamed to `model` in a future release,"
+                " at which point use of `build_fn` will raise an Error instead."
+            )
+        for kwd in kwargs.keys():
+            if "__" not in kwd:
+                raise ValueError(
```

Comment:
The Scikit-Learn docs recommend not doing any validation in `__init__`.

> There should be no logic, not even input validation, and the parameters should not be changed. The corresponding logic should be put where the parameters are used, typically in fit.
> 
> —https://scikit-learn.org/stable/developers/develop.html#instantiation

I think this should go in `BaseWrapper.get_params` before a call to `super().get_params`.

Reply:
That sounds good, I'm fairly certain we will always use `get_params` within `fit`.